### PR TITLE
docs: max_tokens documentation for vision

### PR DESCRIPTION
**Pull request by iloveitaly**
From discord:

https://discord.com/channels/1096466116672487547/1105169591619027055/1219760313520492624

----
📚 Documentation preview 📚: https://griptape--228.org.readthedocs.build//228/

### CODE REVIEW

**Review by andrewfrench — APPROVED** [review]
Thank you @iloveitaly!

File: docs/griptape-framework/engines/image-query-engines.md [hunk 1]
```diff
@@ -5,16 +5,21 @@ The [Image Query Engine](../../reference/griptape/engines/image_query/image_quer
...
 from griptape.drivers import OpenAiVisionImageQueryDriver
 from griptape.engines import ImageQueryEngine
-from griptape.loaders import ImageLoader 
+from griptape.loaders import ImageLoader
 
 driver = OpenAiVisionImageQueryDriver()
 
 engine = ImageQueryEngine(
     image_query_driver=driver,
+    max_tokens=100
```
Comment (andrewfrench) [inline docs/griptape-framework/engines/image-query-engines.md:14]: Whoops, just noticed this. The `max_tokens` parameter should be set when we initialize the driver, not the engine.

**Review by andrewfrench — CHANGES_REQUESTED** [review]

File: docs/griptape-framework/engines/image-query-engines.md (hunk 1 above)
Comment (iloveitaly) [reply docs/griptape-framework/engines/image-query-engines.md]: ah, good catch! Just fixed.

**Comment by emjay07** [comment]
Hey @iloveitaly -- We have updated the defaults for `max_tokens`, so I have integrated this change into our other [Docs Update PR](https://github.com/griptape-ai/griptape-docs/pull/234). Thanks again for the callout!

**Pull request closed without merging**